how-to-handle-history-mode-batch-files.md

```diff
@@ -0,0 +1,97 @@
+# What is History Mode
+
+History mode allows to capture every available version of each record from Fivetran source connectors.
+In order to keep all versions of the records, three new system columns are added to tables with history mode enabled.
+
+
+Column | Type | Description
+--- | --- | ---
+_fivetran_active | Boolean | TRUE if it is the currently active record. FALSE if it is a historical version of the record. Only one version of the record can be TRUE.
+_fivetran_start | TimeStamp | The time when the record was first created or modified in the source.
+_fivetran_end | TimeStamp | The value for this column depends on whether the record is active. If the record is not active, then `_fivetran_end` value will be `_fivetran_start` of the next version of the record minus 1 millisecond. If the record is deleted, then the value will be the same as the timestamp of delete operation. If the record is active, then `_fivetran_end` is set to maximum TIMESTAMP value.
+
+
+### How to Maintain History?  
+
+Following types of files are part of the `WriteHistoryBatchRequest` grpc call. These files need to be processed as described below and in the same order.  
+
+#### `earliest_start_files`
+- In `WriteHistoryBatchRequest`, we pass a new field, `earliest_start_files`. This file will consist of a single record for each primary key and the earliest value of the _fivetran_start value in the incoming batch.
+  For this new file, perform following operations in order:
+    - **Delete overlapping records** : 
+        
+      Delete query to remove any overlapping records where existing `_fivetran_start` is greater than the earliest_fivetran_start timestamp value in the `earliest_start_files` file.
```

Comment:
1. What is the <schema.table> table the DELETE query is run against - source schema? 
2. _earliest_fivetran_start - where is the value obtained from?